lib/hedwig_mopidy/responders/mopidy.ex

```diff
@@ -102,10 +102,11 @@ defmodule HedwigMopidy.Responders.Mopidy do
           playlist
         end
       end
+    send message, "Working on it.."      
     response =
       with  {:ok, :success} <- Tracklist.clear,
-            {:ok, playlist_refs} <- Playlists.get_items(playlist.uri),
-            {:ok, tracks} when is_list(tracks) <- Tracklist.add(playlist_refs |> Enum.map(fn(%Ref{} = r) -> r.uri end)),
+            {:ok, playlist_refs} <- Playlists.get_items(playlist.uri),            
```

Comment:
yeah, but we'll cross that bridge if/when we come to it.
Think of this as an experiment to see if the other one is the real problem.